src/data.rs

```diff
@@ -120,6 +120,14 @@ impl Data {
         self.teams.values()
     }
 
+    pub(crate) fn subteams_of<'a>(
+        &'a self,
+        team_name: &'a str,
+    ) -> impl Iterator<Item = &Team> + 'a {
+        self.teams()
+            .filter(move |t| t.subteam_of().map(|t| t == team_name).unwrap_or_default())
```

Comment:
This seems like it needs to be a recursive check? We want the "grandchild" team as well here.

Reply:
We apparently [do not support subteams of subteams](https://github.com/rust-lang/team/blob/e991f8aa5f8ff0a6ce2f3c7db16385732b735725/src/validate.rs#L160-L164). However, I could make this function work recursively even if the rest of the code base doesn't support this. What do you think?

Reply:
@rylev I think we're going to want subteams of subteams, in the model in which every team is attached somewhere. Doesn't have to be in this commit, though.

Reply:
Added recursive logic. We still need additional support to allow for grandchildren teams, but at least this new feature will handle that when we do.